src/demo/models/guide.js

```diff
@@ -168,7 +170,7 @@ const guides = [
   },
   {
     id: 'creaturesvtrashdemo-predicting-pause2',
-    text: 'But that doesn’t mean they should be removed!',
+    text: `There are lots of marine life in the ocean that don’t look like fish.  But that doesn’t mean they don't belong in the ocean.`,
```

Comment:
Can "marine life" be plural?  It reads as singular here, so I think it would read more naturally as something like: 

"There is a lot of marine life in the ocean that doesn't look like fish."

or 

"There are a lot of marine animals that don't look like animals."

Not sure about exact grammatical rule though.

Reply:
Let's go with:  "There is a lot of marine life in the ocean that doesn't look like fish."

Reply:
Or, perhaps "There is a lot of marine life in the ocean that do not look like fish."

Reply:
I like the first one.  The second one would need to be "There is a lot of marine life in the ocean that *does* not look like fish" or "There *are* a lot of marine life in the ocean that do not look like fish."

Reply:
Ok, I prefer this one for now: "There is a lot of marine life in the ocean that does not look like fish."